doc/xxhash_spec.md

```diff
@@ -43,13 +47,18 @@ However, a given variant shall produce exactly the same output, irrespective of
 ### Operation notations
 
 All operations are performed modulo {32,64} bits. Arithmetic overflows are expected.
-`XXH32` uses 32-bit modular operations. `XXH64` uses 64-bit modular operations.
+`XXH32` uses 32-bit modular operations. `XXH64` and `XXH3` uses 64-bit modular operations.
 
 - `+`: denotes modular addition
+- `-`: denotes modular subtraction
 - `*`: denotes modular multiplication
+    - **Exception:** In `XXH3`, if it is in the form `(u128)x * (u128)y`, it denotes 64-bit by 64-bit normal multiplication into a full 128-bit result.
```

Comment:
👍 
It's interesting, the width of the modular arithmetic may be not obvious, making `*` alone possibly insufficient to describe the operation.

Reply:
After reviewing the rest of the specification, I believe that the propose notification for 128-bit modular arithmetic does the job correctly.